spec/support/shared_examples/participant_actions_withdraw_support.rb

```diff
@@ -9,9 +9,19 @@
   end
 
   it "creates a withdrawn state and makes the profile withdrawn" do
+    mailer = double(SchoolMailer, deliver_later: nil)
+    allow(SchoolMailer).to receive(:fip_provider_has_withdrawn_a_participant).and_return(mailer)
+
     expect { described_class.call(params: given_params) }.to change { ParticipantProfileState.count }.by(1)
     expect(user_profile.participant_profile_state).to be_withdrawn
     expect(user_profile).to be_training_status_withdrawn
+
+    if user_profile.user.participant?
```

Comment:
`user_profile.user`? Should we rename `user_profile` to `participant_profile`?

Reply:
Would that not need a lot of changes? I thought `user_profile` came from https://github.com/DFE-Digital/early-careers-framework/blob/develop/spec/services/participants/withdraw/early_career_teacher_spec.rb

Reply:
Looks like these declaration specs are riddled with `user_profile` being used as an abstraction instead of `participant_profile`, there may be a reason for it. I'd be tempted to follow the same pattern for now.